app/helpers/simple_server_env_helper.rb

```diff
@@ -1,5 +1,6 @@
 module SimpleServerEnvHelper
-  CUSTOMIZED_ENVS = %w[development qa sandbox demo production].freeze
+  CUSTOMIZED_ENVS = %w[development demo qa sandbox production].freeze
```

Comment:
How far reaching is this little constant? I'm wondering if we can easily add real support for our review envs as well while we're here, or whether that's a task for later.

Reply:
Yeah we can always add that later, I'm guessing review falls into the `default` bucket for now.